README.md

```diff
@@ -15,8 +15,12 @@ Build the library using Maven 3.3.9 or newer as shown below:
 git clone https://github.com/linkedin/spark-tfrecord.git
 cd spark-tfrecord
 mvn clean install
-# one can specific the spark version and tensorflow hadoop version
-# mvn clean install -Dspark.version=2.2.1 -Dtensorflow.hadoop.version=1.15.0
+
+# One can specify the spark version and tensorflow hadoop version, for example
+mvn clean install -Dspark.version=2.2.1 -Dtensorflow.hadoop.version=1.15.0
```

Comment:
Since the default spark version is 2.4.6, we can simplify the instruction to build with Spark 2 as `mvn clean install`.